provider/aws/apps.go

```diff
@@ -19,37 +18,24 @@ func (p *AWSProvider) AppGet(name string) (*structs.App, error) {
 	var res *cloudformation.DescribeStacksOutput
 	var err error
 
-	if name == os.Getenv("RACK") {
-		res, err = p.describeStacks(&cloudformation.DescribeStacksInput{
-			StackName: aws.String(name),
-		})
-	} else {
-
-		// try '$RACK-myapp', and if not found try 'myapp'
-		res, err = p.describeStacks(&cloudformation.DescribeStacksInput{
-			StackName: aws.String(os.Getenv("RACK") + "-" + name),
-		})
-
-		if awsError(err) == "ValidationError" {
-			res, err = p.describeStacks(&cloudformation.DescribeStacksInput{
```

Comment:
Yes. An older release completely dropped support for deploying them. Anyone that really needs to stay on that format can use an older Rack release.